Bottom: d6460a9490dc2c31463e3ae0d63e9195368563b4
Top:    c1d1f17ec0fa2d4a9d6d149d6f51232fbe9153b9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:05:32 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
